packages/design-system/src/components/ai-command-bar/command-bar-button.tsx

```diff
@@ -0,0 +1,18 @@
+import { forwardRef, type ComponentProps, type Ref } from "react";
+import { Button } from "../button";
+import { styled, theme } from "../../stitches.config";
+
+type CommandButtonProps = ComponentProps<typeof Button>;
+
+const RoundedButton = styled(Button, {
+  borderRadius: theme.borderRadius["pill"],
+  width: "min-content",
+});
+
+export const CommandBarButton = forwardRef(
+  ({ children, ...props }: CommandButtonProps, ref: Ref<HTMLButtonElement>) => {
+    return <RoundedButton ref={ref} {...props} prefix={children} />;
+  }
+);
```

Comment:
I wonder what this component is for, styled()  is forwarding the ref automatically, doesn't it?

Reply:
children passed to prefix

Reply:
you could as well use prefix, seems like a minor detail and not worth a separate component, unless you think there will be much more logic in it

Reply:
We already wrapped it, for styling, just to make it look better